### PR TITLE
Make external links open in new tab

### PR DESCRIPTION
**Pull request by lyyder**
Links that direct to a different domain are opened in a new tab.

### CODE REVIEW

**Comment by lyyder** [comment]
Currently only menu links added by admin are considered.

File: app/controllers/landing_page_controller.rb [hunk 1]
```diff
@@ -275,7 +275,8 @@ def topbar_props(community, community_customization, request_path, locale_param,
       path_after_locale_change: path,
       search_placeholder: community_customization&.search_placeholder,
       locale_param: locale_param,
-      landing_page: landing_page)
+      landing_page: landing_page,
+      host_with_port: request.host_with_port)
```
Comment (Gnito) [inline app/controllers/landing_page_controller.rb]: You could probably use "railsContext" Location" too.

File: app/view_utils/topbar_helper.rb [hunk 2]
```diff
@@ -127,4 +129,8 @@ def missing_profile_image_path
   def paths
     @_url_herlpers ||= Rails.application.routes.url_helpers
   end
+
+  def link_external?(url, host_with_port)
```
Comment (Gnito) [inline app/view_utils/topbar_helper.rb]: how about ```is_link_external?``` or ```external_link?```

Comment (lyyder) [inline app/view_utils/topbar_helper.rb]: Yeah, but `FeatureFlagHelper` has `feature_enabled?` which takes a feature and `Email` has `email_available?` which takes an email so I planned to follow the naming on those.

Comment (Gnito) [inline app/view_utils/topbar_helper.rb]: Roger

**Pull request merged**